docs/Installing_IML_on_HPC_Storage_Sandbox.md

```diff
@@ -0,0 +1,143 @@
+
+The vagrant file indicates that the lustre network will run on 10.73.20.x. If `Lustre Network 0` is specified for a different IP address, you will need to change its interface to `Not Lustre Network` and update the network for 10.73.20.x to use `Lustre Network 0`. It is very important that Lustre Network 0 is specified on the correct interface; otherwise, creating a filesystem will fail. Make sure that all servers have been updated where appropriate.
+
+## Creating a Filesystem
+To create a filesystem, simply navigate to `Configure->File Systems` and click the `Create` button. Make the following selections:
+- Management Target / MGS -> mds1 (512 MB)
+- Metadata Target / MDS -> mds2
+- Object Storage Targets -> Select ALL OSS nodes
+
+After the selections have been made, click the button to create the filesystem. If you have any issues creating the filesystem there is a good chance that the interface for 10.73.20.x is not assigned to Lustre Network 0. If this happens, stop the filesystem and update the interfaces accordingly. 
+
+## Setting up Clients
+In your vagrant folder, run the following script to prepare both client c1 and c2:
+```
+vagrant sh -c '\
+sudo yum -y install epel-release && \
+sudo yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/managerforlustre/manager-for-lustre/repo/epel-7/managerforlustre-manager-for-lustre-epel-7.repo && \
+sudo yum-config-manager --add-repo http://mirror.centos.org/centos/7/extras/x86_64/ && \
+sudo ed <<EOF /etc/yum.repos.d/mirror.centos.org_centos_7_extras_x86_64_.repo
+/enabled/a
+gpgcheck=1
+gpgkey=http://mirror.centos.org/centos/RPM-GPG-KEY-CentOS-7
+.
+wq
+EOF && \
+sudo yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/managerforlustre/lustre-client/repo/epel-7/managerforlustre-lustre-client-epel-7.repo && \
+sudo yum-config-manager --add-repo https://downloads.hpdd.intel.com/public/e2fsprogs/latest/el7/ && \
+sudo sed -i -e "1d" -e "2s/^.*$/[e2fsprogs]/" -e "/baseurl/s/,/%2C/g" -e "/enabled/a gpgcheck=0" /etc/yum.repos.d/downloads.hpdd.intel.com_public_e2fsprogs_latest_el7_.repo && \
+sudo yum clean metadata && \
+sudo yum -y install lustre-client && \
+sudo reboot' c1 c2
+```
+Both clients are now running the lustre-client software and are ready to be mounted. To mount a client, do the following:
+1. ssh into the client
+```
+vagrant ssh c1
+```
+2. Create a mount directory (it's a good idea to use the same name as the filesystem you created).
+```
+sudo mkdir -p /mnt/fs
+```
+3.  Note the NID for the MGS. You can get this by looking at the lustre network interface in the GUI on the server detail page for mds1 or you can run the following command:
+```
+vagrant ssh mds1
+lctl list_nids
+# 10.73.20.11@tcp # Notice this is the lustre network interface
```

Comment:
Does `lctl list_nids` really return it's results with a `#` in front?

It's actually very important that code and command examples and their output reflect reality as closely as possible to keep confusion to a minimum.  You have to keep in mind that people running these commands might have no prior exposure so even small differences can be alarming.

Reply:
Fixed